include/zephyr/init.h

```diff
@@ -64,16 +64,16 @@ struct init_entry {
 /**
  * @brief Construct a namespaced identifier for SYS_INIT instance
  *
- * @param _name Base unique name
+ * @param name Base unique name
  */
-#define Z_SYS_NAME(_name) _CONCAT(sys_init_, _name)
+#define Z_SYS_NAME(name) _CONCAT(sys_init_, name)
 
 /**
  * @brief Construct a namespaced identifier for a struct init_entry instance
  *
- * @param _entry_name Base unique name
+ * @param entry_name Base unique name
  */
-#define Z_INIT_ENTRY_NAME(_entry_name) _CONCAT(__init_, _entry_name)
```

Comment:
pleasee 9247e8bc44145c88654d54d8b380bc9514e2c514, those underscores were added to deal with some coding guideline violation, need to check if this still applies.

Reply:
I can't see any immediate violation of 5.7 rule in `init.h` any expert on the topic that can review? In fact, I'm not sure what was fixed in 9247e8b regarding init.h (commit message doesn't mention why `_` was added in init.h or device.h...).